docs/how-to-guides/construct-test-data.md

```diff
@@ -0,0 +1,44 @@
+# How to construct test data based on documents
+This guide will help to construct test data based on the provided documents.
+The test data construction process contains three steps:
```

Comment:
I think user may want to know what specific functionalities they can get by reading introduction.

The steps are not user-facing, not suitable to be placed in the doc beginning. If you want to introduce principle behind data gen, you could add a separate section afterwards.